### DISCUSSION TITLE
Move multiple items since collection of organization to other collection

### DISCUSSION THREAD

**Discussion by ymiart** [dani-garcia/vaultwarden#5147] [Q&A]
Hello,

I need move multiple items since collection of organization to other collection.

Currently, I can do it but not in bulk, only item by item. I have many items and it's will long.

It isn't possible since any client (Web and desktop windows).

Thank you very much.

**Comment by VincentVoyer** [comment]
Hi !

The endpoint PUT /ciphers/share already exist, but it's seems to be a bug in the web client.
I proceeded as follows:
1. I select 2 cipher in "All vaults" menu
2. use "Move selected to organisation" action
3. Select an organisation

In webbrowser devtool we can see this result :

![image](https://github.com/user-attachments/assets/b4ee2832-3099-4e7d-a921-9300d3f79438)

body :
![image](https://github.com/user-attachments/assets/5b27438f-4dd1-415e-a656-b1d106efc867)

response :
![image](https://github.com/user-attachments/assets/d592a84a-e7f6-4399-be06-3b08972e617f)

**Comment by VincentVoyer** [comment]
I created an issue : https://github.com/dani-garcia/vaultwarden/issues/5148

**Reply by BlackDex** [reply]
My reply on the ticket (Which i moved to a discussion).

>You can't move Organizational owned ciphers.
You can only move personal items to an organization.
>
>The clients filter out org owned ciphers and send the filtered result. So if you only selected org owned items, this will be the result.
>
>I have tested moving personal items to an org and that works fine.

**Reply by ymiart** [reply]
Hi,

I confirm that the move since personal item to organization in individual and bulk is possible.
I confirm that the move collection in organization to other collection in individual is possible but not in bulk.

Thank you very much.
